elliottlib/bzutil.py

```diff
@@ -344,6 +345,10 @@ def _update_bug_status(self, bugid, target_status):
     def advisory_bug_ids(advisory_obj):
         raise NotImplementedError
 
+    @staticmethod
```

Comment:
making this a bug tracker method allows us to call this nicely in repair-bugs, attach-bugs and remove-bugs 
when we don't necessarily know which bug ids are being passed